src/command.rs

```diff
@@ -182,7 +182,7 @@ impl EccCommand {
     }
 
     pub fn bytes_into(&self, bytes: &mut BytesMut) {
-        bytes.put_slice(&[0x03, 0x00]);
+        bytes.put_slice(&[0x00, 0x00]);
```

Comment:
While this change isn't directly necessary I guess, it was done due to the i2c chips having a different leading command byte compared to the swi chips (0x03 vs 0x77). While we could just keep the i2c byte as a default here, I thought it would be better to keep it as 0's until the operation on line 172 of ecc.rs where the respective command byte is placed at the start of the send buffer. Ultimately it doesn't matter, so this can stay or go depending on your preference.

Reply:
Would it not work better to actually ask the transport layer for the preamble byte(s) right here instead of overwriting it somewhere later on?